source/magpie/group.d

```diff
@@ -175,7 +178,18 @@ public:
         int[] codes = vectorize(levels);
         int[][] rcodes = transposed(dropper(indexLevels, df.indx.row.codes));
         // Simultaneously arranging all the relavent fields using a zip as displayed in the docs
-        auto arrange = zip(levels, codes[1 .. $], data, rcodes).sort!((a, b) => a[1] < b[1]);
+
+        static if(isHomogeneousType)
+            auto sortdata = transposed(data);
+        else
+            auto ref sortdata = data;
```

Comment:
does that even work? Try `alias sortdata = data;`

Reply:
I tested it on the online ide: https://run.dlang.io/is/d8K9sr
Even alias does the job: https://run.dlang.io/is/X9Xx1O

`alias` afaik doesn't allocate the data again so it's indeed a a better option